src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  --box-shawdow: 0 1px 8px 0 #3d4e611a;
+  --small-box-shawdow: 0 3px 8px 0 #3d4e6133;
+  --border: 1px solid #e1e7ed;
+  --box-shawdow-hover: 0 4px 4px 0 #00000040;
+  --background: linear-gradient(0deg, #e1e7ed, #e1e7ed),
+    linear-gradient(180deg, #fff 0%, #f6f 100%);
+}
+
+::-webkit-input-placeholder {
+  color: #3d4e61;
+}
+
+body {
+  margin: 0;
+}
+
+.big__bar {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+
+  height: 70px;
+  margin: 20px 8px;
+  margin-bottom: 0;
+
+  border-radius: 4px;
+  border: var(--border);
+  box-shadow: var(--box-shawdow);
+}
+
+.big_search {
+  padding-left: 26px;
+}
+
+.input_big {
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: #3d4e61;
+
+  width: 90%;
+  height: 90%;
+
+  margin-left: 15px;
+  outline: none;
+  border: none;
+}
+
+.big__bar:hover {
+  box-shadow: var(--box-shawdow-hover);
+}
+
+.big__bar:focus {
+  background: var(--background);
+}
+
+.input_big:focus {
+  font-family: "Avenir-Heavy", sans-serif;
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.small__bar {
```

Comment:
not fixed
You still duplicate a lot of properties
read the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/07629631-a2d6-49af-b2e3-3d5abeb5f9c1)